### PR TITLE
Add concise #inspect to ViewModel

### PR DESCRIPTION
**Pull request by shvetsovdm**
Before (even do not fit the screen):

screen shot 2016-05-13 at 11 44 02 pm

After:

screen shot 2016-05-13 at 11 43 35 pm

to prevent deeper inspect override Cell::Inspect#inspect_blacklist

``` ruby
module Cell
  module Inspect
    def inspect_blacklist
      [:parent_controller]
    end
  end
end
```

### CODE REVIEW

**Comment by timoschilling** [comment]
super!

maybe it could be like this:
```ruby
def inspect(inspect_parent_controller = false)
  parent_controller_s = if @parent_controller && !inspect_parent_controller
```

**Comment by shvetsovdm** [comment]
@timoschilling I do not mind. You think it will be useful?

**Comment by apotonick** [comment]
I like both! @shvetsovdm You could also override `#inspect` in `cells-rails` and add the `parent_controller` code there, since this is a pure Rails-specific feature, only!

**Comment by shvetsovdm** [comment]
@apotonick totally forgot about that cell is framework agnostic. There is no controller and that explains why I failed to spot the place where controllers injected into cell.

Since problem in `#inspect` comes from controllers only, should we close this PR? I will replicate this changes in cells-rails.

**Comment by apotonick** [comment]
I think it's a good idea to have the `inspect` as a framework-agnostic concept (please move to `cell/inspect.rb`), because in other environments such as Roda or Hanami, you might want to "blacklist" unsolicited objects, too?

**Comment by shvetsovdm** [comment]
@apotonick there is any pattern in `cells` that I should use to implement `cell/inspect.rb`?

**Comment by apotonick** [comment]
Yes, a separate file `cell/inspect.rb` and in a separate module, then include that module in `ViewModel`, that should do the trick. Goal is to make your code easily removable (in a good way).

**Comment by shvetsovdm** [comment]
@apotonick modified solution provided

**Comment by shvetsovdm** [comment]
@apotonick ping

**Comment by seanfcarroll** [comment]
Big +1 for this.

File: test/inspect_test.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      @title = title
+    end
+  end
+
+  def build_model
+    InspectTest::FakeModel.new('Title')
+  end
+  # #inspect
+  it do
+    cell = Cell::ViewModel.(model_obj = build_model, options = { genre: "Djent" })
```
Comment (georgemillo) [inline test/inspect_test.rb]: +1 for the Djent reference :D

**Pull request merged**